src/nl/novi/pokemon/WaterPokemon.java

```diff
@@ -0,0 +1,48 @@
+package nl.novi.pokemon;
+
+public class WaterPokemon extends Pokemon {
+    String name;
```

Comment:
name en sound zijn al gedefinieerd in de super klasse (Pokemon). Het heeft daarom weinig nut om ze nog eens in de subklasse (WaterPokemon) te definieren. De subklasse bouwt juist voort op datgene wat de superklasse al biedt.